onchain/rollups/test/foundry/history/History.t.sol

```diff
@@ -237,7 +206,39 @@ contract HistoryTest is Test {
             submitClaim(dapp, epochHash[i], i, i);
         }
 
-        vm.expectRevert(stdError.indexOOBError);
+        vm.expectRevert(History.InvalidClaimIndex.selector);
         history.getClaim(dapp, abi.encode(claimIndex));
     }
+
+    // utility functions
+    function submitClaim(
```

Comment:
Why don't you use `submitClaim()` from `History.sol`? Do you need a custom implementation?

Reply:
Hey Daniel. Yeah it's basically calling `submitClaim()` from history, with the additional benefit to check the event emitted :)